test/integration/scan_test.go

```diff
@@ -296,15 +296,17 @@ func TestScanCreateEmptyProjectName(t *testing.T) {
 }
 
 func TestScanCreate_ExistingApplicationAndExistingProject_CreateScanSuccessfully(t *testing.T) {
+	projectId, projectName := createProject(t, nil, nil)
 	args := []string{
 		"scan", "create",
 		flag(params.ApplicationName), "my-application",
-		flag(params.ProjectName), getProjectNameForScanTests(),
+		flag(params.ProjectName), projectName,
 		flag(params.SourcesFlag), ".",
 		flag(params.ScanTypes), params.IacType,
 		flag(params.BranchFlag), "dummy_branch",
+		flag(params.DebugFlag),
 	}
-
+	defer deleteProject(t, projectId)
```

Comment:
@BenAlvo1  we deleted the defer delete for everything, right?
@miryamfoiferCX , ן think we shou;d remove it

Reply:
but here we create a new project, not using the same as other test, so when will we delete it?

Reply:
@tamarleviCm